server/runtime/src/test/resources/Dockerfile

```diff
@@ -1 +1,35 @@
 FROM jboss/base-jdk:11
+
+# Set the INFINISPAN_SERVER_HOME env variable
+ENV INFINISPAN_SERVER_HOME /opt/jboss/infinispan-server
```

Comment:
Not specific to this PR ... but is there any reason to keep `jboss` anymore? We're moving away from this downstream, so it seems redundant upstream. Also, why not just `/opt/infinispan`?

Reply:
There is no reason to keep this Dockerfile around either, as it is generated programmatically.
I think /opt/infinispan is a good idea.